release.sh

```diff
@@ -19,6 +19,7 @@ export PACKAGING_TOOLS_IMAGE=${PACKAGING_TOOLS_IMAGE:-artifactory.algol60.net/ds
 export RPM_TOOLS_IMAGE=${RPM_TOOLS_IMAGE:-artifactory.algol60.net/dst-docker-mirror/internal-docker-stable-local/rpm-tools:1.0.0}
 export SKOPEO_IMAGE=${SKOPEO_IMAGE:-artifactory.algol60.net/dst-docker-mirror/quay-remote/skopeo/stable:v1.13.2}
 export CRAY_NEXUS_SETUP_IMAGE=${CRAY_NEXUS_SETUP_IMAGE:-artifactory.algol60.net/csm-docker/stable/cray-nexus-setup:0.7.1}
+export CFS_CONFIG_UTIL_IMAGE=${CFS_CONFIG_UTIL_IMAGE:-arti.hpc.amslabs.hpecorp.net/csm-docker-remote/stable/cfs-config-util:5.0.0}
```

Comment:
Do we need to do anything to make sure this Docker image gets packaged up with the hotfix? It isn't in Nexus on the CSM 1.4 system I checked, so I don't think we can assume it will already be on the system.

Reply:
This packages it up in the hotfix tarball: https://github.com/Cray-HPE/csm-hotfixes/blob/ea2439a7445169e61baa028200c941f5230415b7/release.sh#L116-L118

Reply:
Sweet